src/main/resources/db/changelog/changes/v1.0.3/sql/create-list-versions-table.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS list_versions
+(
+    id                      UUID            NOT NULL PRIMARY KEY,
+    name                    VARCHAR(255)    NOT NULL,
+    description             VARCHAR(1024),
+    entity_type_id          UUID            NOT NULL,
+    fql_query               VARCHAR(1024),
+    user_friendly_query     VARCHAR(1024),
+    fields                  VARCHAR[],
+    created_by              UUID            NOT NULL,
+    created_by_username     VARCHAR(64)     NOT NULL,
+    created_date            TIMESTAMP       NOT NULL,
```

Comment:
These all seem like they'd be static values for the whole life of the list and not change from version to version (it only gets created once), so I'd keep it in list_details. On the other hand, updated_by, updated_by_username, and updated_date would all make sense in a list version

Reply:
either way IMO. Who "created" the version, or who "updated" the list (creating the version)